pkg/metrics/tagkeys.go

```diff
@@ -94,6 +97,12 @@ const (
 	// CommitNone is the string value for the commit key indicating that no
 	// commit has been synced.
 	CommitNone = "NONE"
+	// ApplierController is the string value for the applier controller in the multi-repo mode
+	ApplierController = "applier"
+	// RemediatorController is the string value for the remediator controller in the multi-repo mode
+	RemediatorController = "remediator"
+	// SyncerController is the string value for the syncer controller in the mono-repo mode
+	SyncerController = "syncer"
```

Comment:
Do we still need this? The monorepo code has been moved out of this project.

Reply:
It doesn't seem to be referenced anywhere?

Reply:
Good point. Let me remove it.

Reply:
Removed.